src/cfnlint/rules/resources/properties/Inclusive.py

```diff
@@ -46,11 +46,11 @@ def check(self, properties, inclusions, path):
             return matches
 
         for items, p in properties.items_safe(path=path, type_t=(dict)):
-            for k, v in items.items():
+            for k in items.keys():
                 if k in inclusions:
                     for incl_property in inclusions[k]:
-                        if incl_property not in v:
```

Comment:
Wow...  Good catch. Any tests we could add for this?  I'm sure we already had some but I'm surprised the good tests still worked.